src/backend/routers/students.test.ts

```diff
@@ -72,9 +76,10 @@ test("doNotAddDuplicateEmails", async (t) => {
       first_name: "Foos",
       last_name: "Bar",
       email: "foo.bar@email.com",
+      assigned_case_manager_id: seed.para.user_id,
```

Comment:
🐧  somewhat oos but maybe we should move student creation into `src/backend/tests/seed.ts`

Reply:
I'm not sure what you mean by moving to that folder. Most of the tests require the use of `seed`. Also, the `users.test.ts` also contains `seed`.

Reply:
sorry, meant we could add a test student in the seed function instead of re-creating in every test

but oos, not needed for this pr